src/validation.h

```diff
@@ -415,8 +415,6 @@ enum class CoinsCacheSizeState
     OK = 0
 };
 
-using FopenFn = std::function<FILE*(const fs::path&, const char*)>;
```

Comment:
3298c26002a24dd196348b9849e6bc0b2e5e56fd: Instead of moving this twice in the same pull, what about only moving it only once to the final place in one go?

Reply:
Moved once in b3267258b0